lib/flavour_saver/helpers.rb

```diff
@@ -27,6 +27,8 @@ def each(collection)
 
       def if(truthy)
         truthy = false if truthy.respond_to?(:size) && (truthy.size == 0)
+        truthy = false if truthy == 0
```

Comment:
The Numeric class has a ‘#zero?’ method that we should probably use here to cover cases where the value is something like 0.0.

Reply:
Sounds great 👍